libs/community/langchain_community/chat_models/tongyi.py

```diff
@@ -71,8 +71,14 @@ def convert_dict_to_message(
             else HumanMessage(content=content)
         )
     elif role == "assistant":
+        if "tool_calls" in _dict:
+            additional_kwargs = {"tool_calls": _dict.get("tool_calls", [])}
+        else:
+            additional_kwargs = {}
         return (
-            AIMessageChunk(content=content) if is_chunk else AIMessage(content=content)
+            AIMessageChunk(content=content)
+            if is_chunk
+            else AIMessage(content=content, additional_kwargs={**additional_kwargs})
```

Comment:
could we update to also set the new AIMessage.tool_calls param? can see ChatOpenAI for an example

Reply:
@baskaryan fixed according to comments.